README.md

```diff
@@ -126,6 +126,61 @@ Typically, an AKO instance could have been deployed in the `avi-system` namespac
 kubectl delete ns avi-system
 ```
 
+## Upgrade AMKO using *helm*
+
+Follow these steps if you are upgrading from an older AMKO release.
+
+*Step1*
+
+Helm does not upgrade the CRDs during a release upgrade. Before you upgrade a release, run the following command to download and upgrade the CRDs:
+
+```
+helm template ako/amko --version 1.6.1 --include-crds --output-dir <output_dir>
```

Comment:
@swathinsankaran: This command will fail if customer doesn't ako/amko at given version. So I think we can mention command `helm repo update` (if it is not mentioned already) to get an update from ako repo for latest charts available.

Reply:
I have added the repo update command as step1.